index.html

```diff
@@ -175,8 +175,9 @@
     <div class="card">
       <table>
       <tr>
-      <td>
-      		<img src="profile_pictures/<image_name>.jpg" alt="Your name">
+      <td width="200px">
```

Comment:
The `width=200px` attribute was removed from all `td` cells when PR #39 was merged into the main `gh-pages` branch.